examples/CopyWorkspaceSampleScript.ps1

```diff
@@ -211,8 +216,21 @@ Foreach ($dashboard in $dashboards) {
     Foreach ($tile in $tiles) {
         try {
             $tile_id = $tile.id
-            $tile_report_Id = [GUID]($tile.reportId)
-            $tile_dataset_Id = [GUID]($tile.datasetId)
+            if($tile.reportId) {
```

Comment:
Can we be consistent with existing code in styling? There's a space in between the if and opening brace for example in the line # 234 in the same file

Reply:
Updated by running the script through a formatter to apply consistent spacing and bracket alignment